src/logger.ts

```diff
@@ -0,0 +1,30 @@
+import { createLogger, format, transports } from "winston";
+import path from "path";
+import DailyRotateFile from "winston-daily-rotate-file";
+
+const logDir = path.join(__dirname, "logs");
+
+export const logger = createLogger({
```

Comment:
If I'm not mistaken, your logger doesn't pipe its output to stdout when running locally.
In order to avoid having to use both console.logs and custom loggers, what we do is check whether `process.env.NODE_ENV` is "development" and add a transport which pipes the log to stdout. 

Another thing is that we would usually prefer to store the logs in a JSON format in production environments. This allows for easier querying down the line.

Reply:
I added a `NODE_ENV` variable in the environment and also altered the logger to output logs to the console while in development.